telosnetwork/open-block-explorer
# Selling max RAM, Adding CPU/NET

# Fixes #629 #631 

## Test scenarios
sell max RAM:
1. navigate to wallet, select 'resources' to open dialog, select 'sell RAM' tab
2. click max amount and 'sell' button, complete transaction
3. refresh and see 100% usage, 0 available to sell in 'sell RAM' tab

add cpu/net:
1. navigate to wallet, select 'resources' to open dialog, select 'buy cpu/net' tab
2. enter input value for cpu and/or net and click transaction should succeed.
3. refresh and see new cpu/net stake balance(s) reflected on wallet page and resources dialog 

## Checklist:
